components/rating.css

```diff
@@ -0,0 +1,30 @@
+.rating .fa-star{
+    color: var(--lynx-gray-color);
+}
+
+.rating .fa-star:nth-child(1),
+.rating .fa-star:nth-child(2),
+.rating .fa-star:nth-child(3){
+    color: var(--star-color);
```

Comment:
Just a suggestion. Instead of using a child selector, create a class `filled` or something similar and add that class to the stars that you want to color.

Reply:
Done